src/easyscience/fitting/calculators/interface_factory.py

```diff
@@ -3,34 +3,28 @@
 #  SPDX-FileCopyrightText: 2023 EasyScience contributors  <core@easyscience.software>
 #  SPDX-License-Identifier: BSD-3-Clause
 #  © 2021-2023 Contributors to the EasyScience project <https://github.com/easyScience/EasyScience
-
-__author__ = 'github.com/wardsimon'
-__version__ = '0.1.0'
-
-from abc import ABCMeta
 from typing import TYPE_CHECKING
 from typing import Callable
 from typing import List
 from typing import NamedTuple
 from typing import Optional
 from typing import Type
-from typing import TypeVar
 
-_C = TypeVar('_C', bound=ABCMeta)
-_M = TypeVar('_M')
 if TYPE_CHECKING:
-    from easyscience.fitting import Fitter
+    from abc import ABCMeta
+
+    from .. import Fitter
 
 
 class InterfaceFactoryTemplate:
     """
     This class allows for the creation and transference of interfaces.
     """
 
-    def __init__(self, interface_list: List[_C], *args, **kwargs):
-        self._interfaces: List[_C] = interface_list
-        self._current_interface: _C
-        self.__interface_obj: _M = None
+    def __init__(self, interface_list: List[ABCMeta], *args, **kwargs):
+        self._interfaces: List[ABCMeta] = interface_list
+        self._current_interface: ABCMeta
+        self.__interface_obj: None = None
```

Comment:
This is a very bizarre assignment. Both the original and this.
It's probably better/safer to use `Any` here